docs/development/release.md

```diff
@@ -12,11 +12,14 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
 
 #### From Master
 
-If creating a new minor version release from master, create a branch titled `release-MAJOR.MINOR.x` (e.g. release-1.14.x) as a minor version branch from master to allow us to easily backport patches to that version.  Then create a release branch from the minor version branch.
+If creating a new minor version release from master, create a branch titled `release-MAJOR.MINOR.x` (e.g. release-1.14.x) as a minor version base branch from master to allow us to easily backport patches to that version.
+
+Push the `release-MAJOR.MINOR.x` branch to GitHub. (Commits should be even with master at this point.)
+Then create the release branch (e.g. release-1.14.0) from the minor version branch.
 
 #### Backporting
 
-Checkout the minor version branch created in the `From Master` step above, then create a release branch from it.
+Checkout the existing minor version branch created previously in the `From Master` step above, then create a release branch from it.
```

Comment:
Wasn't sure how best to phrase that backporting does not involve doing the `From Master` step at all as a branch should already exist.